BootloaderCommonPkg/Include/Library/SecureBootLib.h

```diff
@@ -78,6 +74,8 @@ DoRsaVerify (
   IN CONST UINT8           *Signature,
   IN       UINT8           *PubKey,
   IN       UINT8           *PubKeyHash      OPTIONAL,
+  IN       UINT8            SignKeySize     OPTIONAL,
```

Comment:
Don't we need additional argument to notify the hash algorithm being used for PubKeyHash itself?